nixos/modules/config/no-x-libs.nix

```diff
@@ -71,7 +72,7 @@ with lib;
       qemu = super.qemu.override { gtkSupport = false; spiceSupport = false; sdlSupport = false; };
       qrencode = super.qrencode.overrideAttrs (_: { doCheck = false; });
       qt5 = super.qt5.overrideScope (const (super': {
-        qtbase = super'.qtbase.override { withGtk3 = false; };
+        qtbase = super'.qtbase.override { withGtk3 = false; withQttranslation = false; };
```

Comment:
TBH I am not sure if this is correct for all cases and right now qt5.qtbase does not build.

Reply:
At least with these changes my system updated normally.